scalability_and_performance/telco_ran_du_ref_design_specs/telco-ran-du-rds.adoc

```diff
@@ -0,0 +1,168 @@
+
+include::modules/telco-ran-node-tuning-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/cnf-tuning-low-latency-nodes-with-perf-profile.adoc#about_irq_affinity_setting_cnf-low-latency-perf-profile[Finding the effective IRQ affinity setting for a node]
+
+include::modules/telco-ran-ptp-operator.adoc[leveloffset=+2]
+
+include::modules/telco-ran-sr-iov-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported SR-IOV devices]
+
+* xref:../../networking/hardware_networks/configuring-sriov-qinq-support.adoc#configuring-qinq-support[Configuring QinQ support for SR-IOV enabled workloads]
+
+include::modules/telco-ran-logging.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../observability/logging/logging-6.0/log6x-about.adoc#log6x-about[About logging]
+
+include::modules/telco-ran-sriov-fec-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://catalog.redhat.com/software/containers/intel/sriov-fec-operator/6017de1669aea3122e6fa15f[SRIOV-FEC Operator for Intel® vRAN Dedicated Accelerator manager container]
+
+include::modules/telco-ran-lca-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc#cnf-understanding-image-based-upgrade[Understanding the image-based upgrade for {sno} clusters]
+
+* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.adoc#ztp-image-based-upgrade-shared-container-partition_shared-container-partition[Configuring a shared container directory between ostree stateroots when using {ztp}]
+
+include::modules/telco-ran-local-storage-operator.adoc[leveloffset=+2]
+
+include::modules/telco-ran-lvms-operator.adoc[leveloffset=+2]
+
+include::modules/telco-ran-workload-partitioning.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/enabling-workload-partitioning.adoc#enabling-workload-partitioning[Workload partitioning]
+
+include::modules/telco-ran-cluster-tuning.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/overview/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities]
+
+include::modules/telco-ran-machine-configuration.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/ztp-reference-cluster-configuration-for-vdu.adoc#ztp-sno-install-time-cluster-config[Recommended {sno} cluster configuration for vDU application workloads].
```

Comment:
This matches the topic map, but not the title of the page. I don't think the topic map entry shows in docs.rh, so maybe worth a change